integration-tests/tests/downloadWarningPage.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { AxiosResponse } from 'axios'
+import { getIntegrationTestEnvironmentVariable } from './utils/getIntegrationTestEnvironmentVariable'
+import crypto from 'crypto'
+import { TriggerEndOfFlowSQSPayload } from './utils/types/sqsPayload'
+import { invokeSQSOperationsLambda } from './utils/aws/invokeSQSOperationsLambdaFunction'
+import { sendRequest } from './utils/request/sendRequest'
+import { pollNotifyMockForDownloadUrl } from './utils/notify/pollNotifyMockForDownloadUrl'
+
+describe('Download pages', () => {
+  const assertDownloadNotFoundResponse = (response: AxiosResponse) => {
+    expect(response.status).toEqual(404)
+    expect(response.data).toEqual('')
+  }
+
+  describe('Download warning page - successful download', () => {
+    let randomId = ''
+    let fileContents = ''
+
+    beforeEach(async () => {
```

Comment:
Following the changes to the lambda in the audit account, we'll need to add a zendeskId here too